tests/compile-fail/data_race/atomic_read_write_race_alt.rs

```diff
@@ -0,0 +1,31 @@
+// ignore-windows: Concurrency on Windows is not supported yet.
+
+use std::thread::spawn;
+use std::sync::atomic::AtomicUsize;
+use std::sync::atomic::Ordering;
```

Comment:
Maybe "atomic_read_na_write" would be clearer as filename, to indicate that the write is *not* atomic. (Same for the other tests.) The "race" suffix is redundant as the folder name already says as much.

When there are multiple tests for the same thing I usually numbered them 1, 2, ...; that seems more scalable than "_alt" suffixes.